### ISSUE TITLE
dev20220311: please send <cr><lf> at line endings of log

### ISSUE THREAD

**Issue by W-Geronius** [wellenvogel/esp32-nmea2000#35]
not really an issue, but a sincere wish to make windows people happy .... it's hard to read like this:

![image](https://user-images.githubusercontent.com/18160302/157934995-67af2e6e-47e5-4c19-98b0-5bda494a4a66.png)

**Comment by wellenvogel** [comment]
I guess putty can do this easily:
https://the.earth.li/~sgtatham/putty/0.76/htmldoc/Chapter4.html#config-crlf

**Comment by W-Geronius** [comment]
true, unfortunately putty is the only program offering this option - are there any drawbacks with <cr><lf> ?

**Comment by W-Geronius** [comment]
thanks for trying to fix this - hmmm ... now however there is an excessive 0x0A:

![image](https://user-images.githubusercontent.com/18160302/161104053-fe706746-62f1-4743-87a2-fcab88f9bcdb.png)

**Comment by wellenvogel** [comment]
nothing fixed until now.
Empty lines had been there already.
The "0d" are coming from the NMEA lines.

**Comment by wellenvogel** [comment]
Should have been there all the time...
